modules/apps/29-fee/keeper/relay.go

```diff
@@ -23,8 +23,15 @@ func (k Keeper) WriteAcknowledgement(ctx sdk.Context, chanCap *capabilitytypes.C
 
 	k.DeleteForwardRelayerAddress(ctx, packetId)
 
-	ack := types.NewIncentivizedAcknowledgement(relayer, acknowledgement)
-	bz := ack.Acknowledgement()
+	ack := channeltypes.Acknowledgement{}
```

Comment:
@AdityaSripal Is this what you meant?

Reply:
update the WriteAcknowledgement API to take in exported.Acknowledgement and then the code will look good